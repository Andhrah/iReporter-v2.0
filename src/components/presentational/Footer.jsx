import React from 'react';

const Footer = () => (
  <footer>
    <p>Â© Andela 2018 All Rights Reserved.</p>
    <p>Designed By Alexandra Collins</p>
  </footer>
);

export default Footer;
