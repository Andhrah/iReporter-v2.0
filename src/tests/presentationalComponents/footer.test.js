import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../../components/presentational/Footer';

describe('Footer  test suite', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper.length).toBe(1);
    expect(wrapper).toMatchSnapshot();
  });
});
