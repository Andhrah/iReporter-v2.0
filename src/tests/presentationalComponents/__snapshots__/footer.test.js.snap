// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer  test suite renders correctly 1`] = `
<footer>
  <p>
    Â© Andela 2018 All Rights Reserved.
  </p>
  <p>
    Designed By Alexandra Collins
  </p>
</footer>
`;
